test/parallel/test-child-process-exec-buffer.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
 const common = require('../common');
-var assert = require('assert');
-var exec = require('child_process').exec;
-var os = require('os');
-var str = 'hello';
+const assert = require('assert');
+const exec = require('child_process').exec;
+const os = require('os');
+const str = 'hello';
 
 // default encoding
 exec('echo ' + str, common.mustCall(function(err, stdout, stderr) {
   assert.ok('string', typeof stdout, 'Expected stdout to be a string');
```

Comment:
These `assert.ok()` calls look wrong to me. Would you mean changing to `assert.strictEqual()` and also switching their order (actual then expected).